biojava-core/pom.xml

```diff
@@ -42,8 +42,19 @@
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
I guess there's no need to set scope=test here, because it's done in parent pom, right?

Reply:
yes, that's right, in the <dependencyManagement> section you just need to set the version and scope once, then sub-modules just need to add  the dependency